### PR TITLE
Add Matomo to Sources

### PR DESCRIPTION
**Pull request by wpxgit**
Adding Matomo Data Source to provide a Data Studio Connector for Matomo Analytics (https://matomo.org/)

### CODE REVIEW

**Comment by wpxgit** [comment]
@googlebot I signed it!

File: sources/matomo.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "MATOMO_ANALYTICS",
```
Comment (pfrisella) [inline sources/matomo.json:2]: The filename and id should match. Either change filename to matomo_analytics or change id to MATOMO

**Review by pfrisella — CHANGES_REQUESTED** [review]

File: sources/matomo.json (hunk 1 above)
Comment (wpxgit) [reply sources/matomo.json]: Thanks, changed the id

**Comment by wpxgit** [comment]
@pfrisella i've made the changes - can you please review? Thanks!

**Review by pfrisella — APPROVED** [review]
LGTM. Thanks for adding!

**Pull request merged**